apps/address-book/app/index.js

```diff
@@ -1,48 +1,13 @@
+import '@babel/polyfill'
```

Comment:
Hmm this seems a little fishy to me. Why do you have to do this?

Reply:
To make it stop complaining about plugin-proposal-class-properties, as in Aragon apps:

https://github.com/aragon/aragon-apps/blob/master/apps/finance/app/src/index.js

However, it might have been a different issue - I retested, that error is gone. I removed these imports.